sentry-core/src/main/java/io/sentry/core/transport/HttpTransport.java

```diff
@@ -41,6 +43,27 @@
 @ApiStatus.Internal
 public class HttpTransport implements ITransport {
 
+  enum DataCategory {
+    All("__all__"),
+    Default("default"), // same as Error
+    Error("error"),
+    Session("session"),
+    Attachment("attachment"),
+    Transaction("transaction"),
+    Security("security"),
+    Unknown("unknown");
+
+    private final String category;
+
+    DataCategory(final @NotNull String category) {
+      this.category = category;
+    }
+
+    public String getCategory() {
+      return category;
+    }
+  }
```

Comment:
`l`: Nice that we have an enum now for this 👏. I am a fan of nested classes or enums. They should only exist to serve their enclosing classes though. Are we sure that this enum only exists in combination with `HttpTransport`? If yes, I think we could even make it private. If not, we could consider making it an enum outside of `HttpTransport`.

Reply:
yes, private here makes sense, good catch.